### PR TITLE
[B2BP-605] - Workflow GitHub Storybook

### PR DESCRIPTION
**Pull request by maurodandrea**
Workflow GitHub for build and deploy NextJS Storybook on AWS S3 and CDN

Depends on #284 

#### List of Changes
create _deploy_storybook.yaml_ file

#### Motivation and Context
Workflow GitHub for build and deploy NextJS Storybook on AWS S3 and CDN

#### How Has This Been Tested?
On GitHub test environment

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [X] My change requires not a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .changeset/quiet-camels-laugh.md
[the remark this answers is not in the document]
Comment (maurodandrea) [reply .changeset/quiet-camels-laugh.md:1]: ok, I remove the ,changeset file

File: .github/workflows/deploy_storybook.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    name: Build and Deploy Storybook
+    runs-on: ubuntu-latest
+    
+    concurrency:
+      group: ${{ github.workflow }}
+      cancel-in-progress: false
+      
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
```
[the remark this answers is not in the document]
Comment (maurodandrea) [reply .github/workflows/deploy_storybook.yaml]: ok, I entered a specific sha with last commit

**Review by christian-calabrese — APPROVED** [review]
LGTM

File: .github/workflows/deploy_storybook.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      cancel-in-progress: false
+      
+    steps:
+      - name: Checkout
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
+        
+      - name: Setup Node
+        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+        with:
+          node-version: "18.17"
```
Comment (kin0992) [inline .github/workflows/deploy_storybook.yaml:38]: **question(non-blocking)**: why hardcode this version?

I think you can use the `node-version-file` parameter, or, even better, re-use a workflow already present in the repository.
Take a look [here](https://github.com/pagopa/b2b-portals/blob/main/.github/actions/setup-node/action.yaml)

Comment (maurodandrea) [reply .github/workflows/deploy_storybook.yaml]: "uses: ./.github/actions/setup-node" in new commit

**Review by kin0992 — APPROVED** [review]

**Pull request merged**